### ISSUE TITLE
Sidebar Follow Up

### ISSUE THREAD

**Issue by cschieder** [kubermatic/dashboard#5345]
- [x] unfold/fold animations don't feel smooth, do they have easing? 
- [x] Access menu point doesn't have animations
- [x] remove hover effect on active menu point (on click nothing happens as it cannot be closed/is already active, so no hover makes sense) 
- [x] Have resources be unfolded by default
- [x] Increase bottom and top padding to 10px on sub menu point (makes each subpoint larger) 
- [x] Add headlines to cards after #4587
- [x] Remove commented code for dynamic tabs that was updated in [Project sidebar redesign #5211](https://github.com/kubermatic/dashboard/pull/5211/files#diff-aac70c9d6a5826ccac855034fa8764e87ed7905b89af26eaa7ba3d022ab522f8R23)

**Comment by ahmadhamzh** [comment]
i think the last point is not related to the sidebar so maybe we fix it in separate PR and for the first five point will be fix in this PR : https://github.com/kubermatic/dashboard/pull/5308

**Comment by Waseem826** [comment]
@ahmadhamzh @KhizerRehan added another task in issue description:

> Remove commented code for dynamic tabs that was updated in [Project sidebar redesign #5211](https://github.com/kubermatic/dashboard/pull/5211/files#diff-aac70c9d6a5826ccac855034fa8764e87ed7905b89af26eaa7ba3d022ab522f8R23)

**Comment by ahmadhamzh** [comment]
the `Add headlines to cards after` will be fixed in #5371
